src/controls/arrow.js

```diff
@@ -27,14 +32,22 @@ export default class Arrow extends Component {
   }
 
   render() {
-    const { prevSlide, nextSlide, direction } = this.props;
+    const { prevSlide, nextSlide, direction, customArrow } = this.props;
+    let arrowComponent = null;
+    let buttonClass = 'carousel-arrow-default';
+
+    if (customArrow) {
+      buttonClass = customArrow.className ? customArrow.className : '';
+      arrowComponent = direction === 'left' ? customArrow.left : customArrow.right;
+    }
 
     return (
       <button
```

Comment:
Not your change, but this `<button>` should have `<button type='button'>` on it since the default in HTML is `type="submit"` which is never what this component wants.